explorer.md

```diff
@@ -2,3 +2,29 @@
 permalink: /explorer.html
 ---
 
+# Loc-I Explorer
+
+Test site:
+[http://loci-integration-app-test.s3-website-ap-southeast-2.amazonaws.com](http://loci-integration-app-test.s3-website-ap-southeast-2.amazonaws.com)
+
+## What
+
+A visual explorer that interfaces with the Loc-I reference Integrated Cache APIs. This tool will enable users to visualise and interactively explore (by clicking on links) and searching the graph of interconnected ontologies (concepts), Loc-I datasets, spatial features within them and linksets that describe spatial relationships between them (i.e. by location, text labels, codes). 
```

Comment:
Yikes - 'the Loc-I reference Integrated Cache APIs' 
maybe 'cached versions of the Loc-I reference datasets' or similar

Reply:
there will be multiple caches, however, for the Loc-I project, there will be a reference cache loaded with a reference set of datasets/linksets/ontologies. 

but yes, `Loc-I reference Integrated Cache APIs` is a bit of a mouthful... any other alternatives?

Reply:
Loci-I Integration API?

Reply:
I'd remove 'integrated' and leave 'reference'

Reply:
I find reference very confusing, does it mean it is a the place I should go to get the information, what about the LDAPIs? The Loc-I APIs main feature is the ability to integrate by virtue of the integrated linked data db, the collated datasets within, the linksets, and potentially other proxies directly to the LDAPIs. Through integration it allows cross cutting queries that span different spatial linked datasets (i.e the LDAPIs).

Reply:
agree that these names are not well developed. we'll need to discuss these as part of a broader discussion I think.